### PR TITLE
Entrypoint script and hosts file

### PR DESCRIPTION
**Pull request by wesleymonte**

### CODE REVIEW

**Review by joivo — APPROVED** [review]

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+# This script goal is to generate the Ansible hosts and ansible.cfg files
```
Comment (raonismaneoto) [inline install.sh]: and to run the ansible script as well, right?

**Review by raonismaneoto — CHANGES_REQUESTED** [review]

File: install.sh (hunk 1 above)
Comment (wesleymonte) [reply install.sh]: Yes. Updated

**Review by raonismaneoto — APPROVED** [review]

File: hosts.conf [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Saps Deploy
+## Ansible user name on the hosts where the Saps component will be deployed (should be the same in all hosts)
```
Comment (thiagomanel) [inline hosts.conf:2]: why do we need so many private keys in this declaration? Do we want to allow the user to use different keys? I guess the user would use the same key here, wouldn't?

**Review by thiagomanel — CHANGES_REQUESTED** [review]
a question

File: hosts.conf (hunk 2 above)
Comment (wesleymonte) [reply hosts.conf:2]: You're right. I modified it to use the same key for all hosts.

**Review by thiagomanel — APPROVED** [review]
lgtm

**Pull request merged**